crates/workspace/src/pane_group.rs

```diff
@@ -944,7 +952,51 @@ mod element {
                 child.element.paint(cx);
             }
 
+            let overlay_opacity = WorkspaceSettings::get(None, cx)
+                .active_pane_modifiers
+                .inactive_opacity
+                .and_then(|val| (val <= 1.).then_some(val));
+
+            let mut overlay_background = cx.theme().colors().editor_background;
+            if let Some(opacity) = overlay_opacity {
+                overlay_background.fade_out(opacity);
+            }
+
+            let overlay_border = WorkspaceSettings::get(None, cx)
+                .active_pane_modifiers
+                .border_size
+                .and_then(|val| (val >= 0.).then_some(val));
+
             for (ix, child) in &mut layout.children.iter_mut().enumerate() {
+                if overlay_opacity.is_some() || overlay_border.is_some() {
+                    let overlay_bounds = Bounds {
+                        origin: child
+                            .bounds
+                            .origin
+                            .apply_along(Axis::Horizontal, |val| val + Pixels(1.)),
```

Comment:
Not sure I understand those 1.0 pixel dances, can you elaborate a bit what's this for?

Reply:
added a comment in the code, but tl;dr is to accomodate the divider, which is painted on the right of the previous pane, so the overlay has to be painted in origin+1px and the size has to be width-1px to leave that single pixel for the divider